manifests/input.yaml

```diff
@@ -651,28 +661,83 @@ work/input/make_ryf/make_ryf.py:
   hashes:
     binhash: 9fbb6a9f6bec50ccb2afafecae9a5ff7
     md5: c5ef2aeff9516ec76c80e76aef08d352
+work/input/mask_table.464.48x40:
+  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/025deg/mom/mask_table.464.48x40
+  hashes:
+    binhash: 339b390c248283a1bc8113dadf3d4242
+    md5: 7a2fcdfbb9b331161c7d8874e2b7e6f9
+work/input/meshes/JRA55do-ESMFmesh.nc:
+  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/025deg/share/meshes/JRA55do-ESMFmesh.nc
```

Comment:
I dont think this file is specific to 0.25 degree right? Its the same file as 1degree? Can we put it in a common folder?

Reply:
Update input and restart yaml files in this commit https://github.com/COSIMA/MOM6-CICE6/commit/058e5f7ee3ae4fd031bc1e9f87ae1e74ad263382